app/services/employees/csv_headers_validator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Employees
+  class CsvHeadersValidator < ApplicationService
+    attr_reader :attachment
+
+    def initialize(attachment)
+      super()
+      @attachment = attachment
+    end
+
+    def call
+      csv_headers = []
+
+      CSV.foreach(attachment, **Employees::Props::CSV_OPTIONS) do |row|
+        csv_headers = row.headers
```

Comment:
it could be just valid_headers = false
in CSV for Each valid_headers = (Employees::Props::REQUIRED_USER - row.headers).empty?
out foreach valid_headers